# 机器人组血量

# 英雄机器人1号
int32 hero1
# 工程机器人2号
int32 engineer2
# 步兵机器人3号
int32 standard3
# 步兵机器人4号
int32 standard4
# 步兵机器人5号
int32 standard5
# 哨兵机器人7号
int32 sentry7
# 前哨站
int32 outpost
# 基地
int32 base